example/deployments/sit.tfvars

```diff
@@ -0,0 +1,49 @@
+buckets = {
+  internal = {
+    name = "cumulus-sit-internal"
+    type = "internal"
+  }
+  private = {
+    name = "cumulus-sit-private"
+    type = "private"
+  },
+  protected = {
+    name = "cumulus-sit-protected"
+    type = "protected"
+  },
+  protected-2 = {
+    name = "cumulus-sit-protected-2"
+    type = "protected"
+  },
+  public = {
+    name = "cumulus-sit-public"
+    type = "public"
+  }
+}
+s3_replicator_config = {
+  source_bucket = ""
+  source_prefix = ""
+  target_bucket = ""
+  target_prefix = ""
+}
+system_bucket="cumulus-sit-internal"
+
+launchpad_api = "https://api.launchpad.nasa.gov/icam/api/sm/v1"
+launchpad_certificate = "launchpad.pfx"
```

Comment:
Does this actually exist in the crypto dir along with the ems keys?

Reply:
Since creating the PR, I've uploaded the keys and created (and then corrected) the TEA JWT secrete. Build just passed, I think everything that should be there is there now.